cql3/statements/update_statement.cc

```diff
@@ -179,9 +179,7 @@ void update_statement::add_update_for_key(mutation& m, const query::clustering_r
 }
 
 modification_statement::json_cache_opt insert_prepared_json_statement::maybe_prepare_json_cache(const query_options& options) const {
-    sstring json_string = with_linearized(_term->bind_and_get(options).data().value(), [&] (bytes_view value) {
-        return utf8_type->to_string(bytes(value));
-    });
+    sstring json_string = utf8_type->to_string(to_bytes(_term->bind_and_get(options)));
```

Comment:
Note: we'll eventually have to fragment json too.

Reply:
The main problem here is that sstring itself contiguous. We will have to replace it with something.

Reply:
```c++
class json {
    managed_bytes _payload;
public:
    ...
};
```